manifests/Lexikos/AutoHotKey/1.1.32.00.yaml

```diff
@@ -0,0 +1,15 @@
+Id: lexikos.autohotkey
```

Comment:
The package ID should be `Lexikos.AutoHotkey`. Your folder name should be `AutoHotkey` as well, following AHK's own casing.

Reply:
Fixed in new commit, just pushed and waiting on Pipelines